src/renderer/gl_engine/tvgGlGpuBuffer.cpp

```diff
@@ -60,6 +72,12 @@ void GlGpuBuffer::unbind(Target target)
     GL_CHECK(glBindBuffer(static_cast<uint32_t>(target), 0));
 }
 
+GlStageBuffer::GlStageBuffer()
```

Comment:
Suggested one line:  "GlStageBuffer::GlStageBuffer() : mVao(0), mGpuBuffer(new GlGpuBuffer)"

Reply:
I changed these property initialization to single line. 
> BTW, I think thorvg really needs to define its own coding style, and the clang format configuration would be very helpful

Reply:
ok, thanks. let's see how we can enhance the policy around 1.0 release